### PR TITLE
Added support for MKR1000 board

### PR DESCRIPTION
**Pull request by fleshin**
Hi,
I am able to flash blinky1.go to MKR1000 and MKR1010 with this code. It is based on the nano-iot33 code.

### CODE REVIEW

File: targets/arduino-mkr1000.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "inherits": ["atsamd21g18a"],
+    "build-tags": ["sam", "atsamd21g18a", "arduino_mkr1000"],
```
Comment (aykevl) [inline targets/arduino-mkr1000.json]: This was recently changed in #1490.

```suggestion
    "build-tags": ["arduino_mkr1000"],
```

**Review by aykevl — COMMENTED** [review]
This looks good! Just one comment regarding a recent change.

**Comment by fleshin** [comment]
Thanks! Used the 0.15 as the base, sorry about that. Your change was applied.

**Comment by aykevl** [comment]
Can you perhaps rebase on the dev branch? There are some CI failures that should be fixed after rebasing, see https://github.com/tinygo-org/tinygo/pull/1497 for details.

**Comment by fleshin** [comment]
I think this worked. All the checks passed.

**Comment by aykevl** [comment]
Thank you! I have merged this PR.

**Pull request merged**